modules/self_managed_nodes/win_cm_vpc_cni.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  aws_auth_configmap_data = {
+    enable-windows-ipam = "true"
+  }
+}
+
+resource "kubernetes_config_map" "amazon-vpc-cni" {
+  count = var.create_aws_auth_configmap ? 1 : 0
```

Comment:
Able to add a variable to turn on support for Windows nodes? Then we only set the workers IAM role and create this resource when windows support is needed.

Reply:
Updated as suggested.